src/FiltrationGroupComponent/FiltrationGroupComponent.tsx

```diff
@@ -8,7 +8,16 @@ import "./FiltrationGroupComponent.scss";
 
 type Props = StateProps & CallbackProps;
 
+interface StateDefaultProps {
+	horizontalBracket: [];
```

Comment:
Наверное тут так надо?
```ts
interface PositionTop {
  positionTop: number;
}
```
```suggestion
	horizontalBracket: PositionTop[];
```

А возможно и просто
```suggestion
	horizontalBracket: number[];
```
Зачем собирать объект из одного свойства?

Reply:
согласен объект тут лишний